src/components/menu-surface/menu-surface.scss

```diff
@@ -14,4 +14,5 @@
 .mdc-menu-surface {
     width: var(--menu-surface-width, auto);
     max-height: 100%;
+    position: inherit;
```

Comment:
I'm not sure I understand what this does 🤔

That is, I understand that it inherits the value of the `position` style from the parent (or closest ancestor where it's set perhaps, not sure), but I don't understand what doing that accomplishes? 🙂

Reply:
Without this, the `.mdc-menu-surface` gets static positioning, which makes the portal get a size of zero.

Reply:
So I say inherit to remove the static positioning that `mdc` adds somewhere deep down in its internal code

Reply:
Static positioning makes it get a size of zero? That's odd 🤔

My thought was that it would be safer to set a fixed value, like `relative`, or `absolute`, because I assumed we wouldn't want it to inherit `fixed` for example. But maybe it actually needs to have the same value not to break… Whatever, if it works it works. Leave it in 😄

Reply:
And resolve this if you're happy with that reply 😛

Reply:
I was considering setting it to `relative` but decided against as inherit would ensure that it gets what's used by the parent. 

Then I press the resolve button :D

Reply:
Ok, if `relative` works, I think we should set it to that. Letting the child inherit `absolute` from the parent seems very likely to break things… 🙂

Reply:
I changed this to `relative`.